packages/botonic-plugin-text-classification/README.md

```diff
@@ -0,0 +1,36 @@
+# Botonic Plugin Text Classification
+
+## What does this plugin do?
+
+Botonic Plugin Text Classification uses the trained models defined by the user to classify the input text.
+
+After detecting the locale of the text, and using the corresponding model to predict the class, extra information about the intent of the user is added to the input message.
```

Comment:
After detecting the text locale, and using the relevant model to predict the class, extra information about the user intent is added to the input message.